database/migration/src/main/scala/cromwell/database/migration/liquibase/LiquibaseUtils.scala

```diff
@@ -21,16 +22,18 @@ object LiquibaseUtils {
     * @param jdbcConnection A jdbc connection to the database.
     */
   def updateSchema(settings: LiquibaseSettings)(jdbcConnection: Connection): Unit = {
-    val liquibaseConnection = newConnection(jdbcConnection)
-    try {
-      val database = DatabaseFactory.getInstance.findCorrectDatabaseImplementation(liquibaseConnection)
-      database.setDatabaseChangeLogLockTableName(settings.databaseChangeLogLockTableName.toUpperCase)
-      database.setDatabaseChangeLogTableName(settings.databaseChangeLogTableName.toUpperCase)
+    mutex.synchronized {
```

Comment:
Do you really need `mutex`?

It seems like `this.synchronized { ... }` would accomplish the same thing (just as there is only ever one `mutex`, there is only ever one `LiquibaseUtils`)

Reply:
See comment below.